From: Henning Westerholt <hw@gilawa.com>
Date: Wed, 14 Dec 2022 08:38:12 +0000
Subject: [PATCH] utils/kamcmd: add missing USE_READLINE define for pkg-config
 build, related to #3284

(cherry picked from commit a70b6bf09c73d961ebea1947216757fafb951c24)
---
 utils/kamcmd/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/kamcmd/Makefile b/utils/kamcmd/Makefile
index f12306b..1fd923d 100644
--- a/utils/kamcmd/Makefile
+++ b/utils/kamcmd/Makefile
@@ -80,6 +80,7 @@ endif
 # detect libreadline
 ifneq ($(BUILDER),)
 	DEFS += $(shell $(BUILDER) --cflags readline)
+	DEFS+=-DUSE_READLINE
 	LIBS += $(shell $(BUILDER) --libs readline)
 	use_readline := 1
 else
